include/z64actor.h

```diff
@@ -161,14 +161,16 @@ typedef struct ActorShape {
 //
 #define ACTOR_FLAG_15 (1 << 15)
 
-//
-#define ACTOR_FLAG_16 (1 << 16)
+// Player automatically accepts a Talk Offer without needing to press the A button.
+// Player still has to meet all conditions to be able to accept a talk offer (for example, being in range).
+#define ACTOR_FLAG_TALK_OFFER_AUTO_ACCEPT (1 << 16)
 
 //
 #define ACTOR_FLAG_17 (1 << 17)
 
-//
-#define ACTOR_FLAG_18 (1 << 18)
+// When locked onto an actor with this flag set, the C-Up button can be used to talk to this actor.
+// A C-Up button labeled "Navi" will appear on the HUD when locked on.
+#define ACTOR_FLAG_TALK_WITH_C_UP (1 << 18)
```

Comment:
does Link talk to the actor, or talk to navi about the actor?

Reply:
I suppose this is only used for navi messages, where in-game Link talks to navi but this is implemented as "talking" with the actor. The flag name seems fine but maybe the description should be something along the lines of "can check this actor with Navi" instead of "can talk to this actor"?

Reply:
Yeah, it is literally talking to the actor with c-up. The elf_msg actor(s) just have text with blue font that makes it look like Navi is talking.
I tried to get the point across that its navi related with the following line.

Reply:
I see, it seemed contradictory to me instead, or maybe that the C-Up "Navi" is a bug. Maybe could use something like "as if checking this actor with Navi" or something but idk

Reply:
`// A C-Up button labeled "Navi" will appear on the HUD when locked on which indicates the actor is being checked with Navi`
?

Reply:
"is being checked" -> "can be checked" but works for me

Reply:
I had the exact same question as cadmic did after reading the updated comment, but I don't know how to improve it

Reply:
I guess just add even more explanation:
```
// With this flag Player talks directly to the actor with C-Up. It is expected that the dialog that follows should seem like
//  it is coming from Navi, even though she is not involved at all with this interaction.
```
?

Reply:
sounds good, just
"should seem like" -> "seems like"

Reply:
sorry i made an edit after originally posting, does this still need changing

Reply:
sounds good